### ISSUE TITLE
Music Computer

### ISSUE THREAD

**Issue by sunnysideofthescreen** [cloyne/common-electronics#14]
This computer is a bit neglected. It should be documented better, made more accessible (better desk area) and its functionality should be checked. Collab with music room manager? I think Mike Zhong (Fall 2016 common elect manager) would be great at this!

**Comment by mitar** [comment]
What is Mike's GitHub username?

**Comment by sunnysideofthescreen** [comment]
not sure!

**Comment by clohard** [comment]
House got new Korg synth which would be the perfect addition to the music room workstation and computer. Reached out to Storm Weiner to see if he would like to help setup.

**Comment by sunnysideofthescreen** [comment]
yeah this project had momentum a while ago (Spring 16?). Brian Dellara was interested in helping back then, he might have insights. 

-kel
